com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -177,6 +182,26 @@ internal set
         /// </summary>
         public bool AutoObjectParentSync = true;
 
+        public InterestSettings InterestSettingsOverride;
+        public InterestSettings InterestSettings
```

Comment:
There's a decent amount of code to expose a global InterestSettings asset, and per-object overrides, but InterestSettings currently has no actual content. Is there already a plan for how this will be used?